Routes/cart.js

```diff
@@ -0,0 +1,47 @@
+const bodyParser = require('body-parser')
+const express = require('express')
+var router = express.Router()
+// const cors = require("cors")
+const {Cart} = require("../models/Cart.model.js")
+const { Product } = require('../models/Product.model.js')
+
+// router.use(cors())
+router.use(bodyParser.json())
+
```

Comment:
You can remove line 1, 4, 8, 9, as we have already used bodyParser and cors in index.js, we won't need it here again